ods_ci/tests/Tests/100__deploy/130__operators/140__serverless_operator/140__serverless_operator.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library         Collections
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Resource        ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup      Suite Setup
+Suite Teardown   Suite Teardown
+
+
+*** Variables ***
+${SERVERLESS_APPNAME}  Red Hat OpenShift Serverless
+${SERVERLESS_OPERATOR_NAME}    Red Hat OpenShift Serverless
+${KNATIVESERVING_NS}    knative-serving
+${ISTIO_NS}     istio-system
+
+
+*** Test Cases ***
+Validate DSC creates all Serverless CRs
+    [Documentation]  The purpose of this Test Case is to validate the creation
+    ...    of Serverless Custom Resources
+    [Tags]  Operator    ODS-2600
+    Assign Vars According To Product    ${PRODUCT}
+    Check And Install Operator in Openshift    ${SERVERLESS_APPNAME}    ${SERVERLESS_OPERATOR_NAME}
+    Check And Install Operator in Openshift    ${OPERATOR_APPNAME}    ${OPERATOR_NAME}
```

Comment:
so you would install RHOAI from OperatorHub right? It won't work with a specific operator version, unless:
1. we test on released version only
2. we test on a cluster where Stage Errata has been configured

Reply:
We will trigger mostly on a cluster with a specific version installed.

Reply:
I think the test should expect RHOAI is installed or fail/skip if it doesn't, but not install RHOAI by itself, specially if this test is run in one of the common quality gates (Smoke, Sanity, ...)

Reply:
Ok I remove the check and install invokations